scripts/deploy-k8s-components.sh

```diff
@@ -2,10 +2,10 @@
 set -ex
 
 ROOT_DIR=$(dirname $0)/..
-NAMESPACE=${NAMESPACE:-examples}
+NAMESPACE=${NAMESPACE:-object-detection}
 
 helm upgrade --install --timeout 600 --wait --debug \
-${NAMESPACE}-videoprocessor \
+${NAMESPACE} \
```

Comment:
Revert this change.

Reply:
made a separate folder for original scripts

Reply:
This change still needs to be reverted.